manifests/service.pp

```diff
@@ -41,6 +65,7 @@
   $id                  = $title,
   $port                = undef,
   $service_name        = $title,
+  Hash $service_config_hash = {},
```

Comment:
Can you please align the `=` chars and enforce the structure of the Hash?

Reply:
Alignment done, enforcement delayed per our slack conversation